libcosim/doxygen/libcosim/0.7.0/classcosim_1_1model__uri__sub__resolver.html

```diff
@@ -100,7 +100,7 @@
 </table>
 <a name="details" id="details"></a><h2 class="groupheader">Detailed Description</h2>
 <div class="textblock"><p>An interface for classes that resolve model URIs of one or more specific URI schemes. </p>
-<p>Examples could be a file URI resolver which handles URIs like <code><a href="file:///models/my_model.fmu">file:///models/my_model.fmu</a></code> or an FMU-proxy URI resolver which handles URIs like <code>fmu-proxy://sim.example.com:6345</code>.</p>
+<p>Examples could be a file URI resolver which handles URIs like <code><a href="file:///models/my_model.fmu">file:///models/my_model.fmu</a></code> or an proxyfmu URI resolver which handles URIs like <code>proxyfmu://localhost:9090?file=models/my_model.fmu</code>.</p>
```

Comment:
Don't think we should alter v0.7.0, it will be overwritten when we upload doxygen doc for libcosim/libcosimc for 0.8.2 anyway

Reply:
Didn't notice that this was doxygen stuff..